tukaan/fonts/font.py

```diff
@@ -151,7 +151,6 @@ def config(
         if size:
             if isinstance(size, float):
                 size = round(size)
-            size = -size  # Negative value means points in Tk, but pixels in Tukaan. Invert it
```

Comment:
Maybe revert this, and use a global tkfont instance to convert points to pixels in `get_font_options_for_text_item`